### DISCUSSION TITLE
Make image uploads simple via the budibase image component

### DISCUSSION THREAD

**Discussion by rpmcoding** [Budibase/budibase#7932] [Ideas]
Sometimes on a screen, we will want to include an image.  The budibase image component currently only allows you to enter an image url, not upload an image.  So it's a faff to go upload it somewhere else and get the url.

Add the ability to upload an image also.  It can be hosted on the same server for ease, and/or add integration with S3/cloudfront.  Ability to optionally compress the image also would be helpful.

**Comment by melohagan** [comment]
Definitely agree! There is a feature request open here: https://github.com/Budibase/budibase/issues/5285
